sql/updates/world/master/2016_11_xx_00_world.sql

```diff
@@ -0,0 +1,335 @@
+(52409, 0, 6, 53485, 1003.07, -66.4913, 55.4067, 0, 8, 0),
+(52409, 0, 6, 53485, 1002.72, -40.7431, 55.4063, 0, 8, 0),
+(52409, 0, 6, 53485, 1003.44, -74.0243, 55.4063, 0, 8, 0),
+(52409, 0, 6, 53485, 999.755, -98.4792, 55.4426, 0, 8, 0),
+(52409, 0, 6, 53485, 1000.81, -14.5069, 55.4566, 0, 8, 0),
+(52409, 0, 6, 53485, 1005.49, -86.4566, 55.4275, 0, 8, 0),
+(52409, 0, 6, 53485, 1006.1, -27.3681, 55.4277, 0, 8, 0),
+(52409, 0, 6, 53485, 1005.8, -8.81771, 55.4672, 0, 8, 0),
+(52409, 0, 6, 53485, 1012.09, -97.5122, 55.457, 0, 8, 0),
+(52409, 0, 6, 53485, 1009.56, -108.161, 55.4697, 0, 8, 0),
+(52409, 0, 6, 53485, 1014.15, -17.3281, 55.4629, 0, 8, 0),
+(52409, 0, 6, 53485, 1012.7, -4.83333, 55.605, 0, 8, 0),
+(52409, 0, 6, 53485, 1018.29, -117.833, 55.4471, 0, 8, 0),
+(52409, 0, 6, 53485, 1019.6, 7.7691, 55.447, 0, 8, 0),
+(52409, 0, 6, 53485, 1024.45, -8.13889, 55.447, 0, 8, 0),
+(52409, 0, 6, 53485, 1024.91, -106.852, 55.4471, 0, 8, 0),
+(52409, 0, 6, 53485, 1023.05, -128.257, 55.4471, 0, 8, 0),
+(52409, 0, 6, 53485, 1023.83, 12.9774, 55.447, 0, 8, 0),
+(52409, 0, 6, 53485, 1035.56, -114.156, 55.4471, 0, 8, 0),
+(52409, 0, 6, 53485, 1035.91, 0.909722, 55.447, 0, 8, 0),
+(52409, 0, 6, 53485, 1032.48, 13.2708, 55.4469, 0, 8, 0),
+(52409, 0, 6, 53485, 1035.17, -125.646, 55.4471, 0, 8, 0),
+(52409, 0, 6, 53485, 1049.74, -118.396, 55.5661, 0, 8, 0),
+(52409, 0, 6, 53485, 1049.33, 5.0434, 55.4633, 0, 8, 0),
+(52409, 0, 6, 53485, 1052.59, -120.562, 55.4563, 0, 8, 0),
+(52409, 0, 6, 53485, 1055.34, 5.06771, 55.4471, 0, 8, 0),
+(52409, 0, 7, 53231, 1026.86, 5.89583, 55.357, 4.90448, 8, 0),
+(52409, 0, 7, 53231, 1027.31, -121.746, 55.3708, 1.36127, 8, 0),
+(52409, 0, 8, 53872, 787.639, -59.9396, 86.2755, 6.20131, 8, 0),
+(52409, 0, 8, 53875, 787.285, -68.8034, 94.0955, 3.33302, 8, 0),
+(52409, 0, 8, 53876, 788.131, -50.6921, 94.0005, 2.96047, 8, 0);
+
+-- Ragnaros
+UPDATE `creature_template` SET `difficulty_entry_1`=53797, `difficulty_entry_2`=53798, `difficulty_entry_3`=53799, `speed_walk`=0.357143, `InhabitType`=4, `ScriptName`='boss_ragnaros_firelands', `VerifiedBuild`=0 WHERE `entry`=52409;
+UPDATE `creature_template` SET `minlevel`=88, `maxlevel`=88, `faction`=2234, `speed_walk`=0.357143, `speed_run`=1.42857, `BaseAttackTime`=1500, `unit_flags2`=134219776, `type_flags2`=128, `lootid`=53797, `InhabitType`=4, `flags_extra`=1, `VerifiedBuild`=0 WHERE `entry`=53797;
+UPDATE `creature_template` SET `minlevel`=88, `maxlevel`=88, `faction`=2234, `speed_walk`=0.357143, `speed_run`=1.42857, `BaseAttackTime`=1500, `unit_flags2`=134219776, `type_flags2`=128, `lootid`=53798, `InhabitType`=4, `flags_extra`=1, `VerifiedBuild`=0 WHERE `entry`=53798;
+UPDATE `creature_template` SET `minlevel`=88, `maxlevel`=88, `faction`=2234, `speed_walk`=0.357143, `speed_run`=1.42857, `BaseAttackTime`=1500, `unit_flags2`=134219776, `type_flags2`=128, `lootid`=53799, `InhabitType`=4, `flags_extra`=1, `VerifiedBuild`=0 WHERE `entry`=53799;
```

Comment:
You can reduce the number of `UPDATE` lines by merging lines with identical content (except for entry). Simply put
```sql
WHERE `entry` IN (53797,53798,53799);
```
at the end of line 104 and remove 105+106. The same goes for any grouped lines below _where the updates are identical for all the fields_.